scripts/prepare_sector_network.py

```diff
@@ -497,6 +497,9 @@ def add_carrier_buses(n, carrier, nodes=None):
         capital_cost=capital_cost,
     )
 
+    if carrier == "gas":
+        return
```

Comment:
```suggestion
```

This would prohibit the use of fossil gas, wouldn't it? 

Since the store is cyclic, where would the model source fossil gas?